.github/workflows/clang-format.yml

```diff
@@ -1,6 +1,7 @@
 name: Clang Format Lint
 
 on:
+  workflow_dispatch:
```

Comment:
This should not be in this PR. You also have multiple commits which are from your older PR, right?

Reply:
Yes sir. I don't know why it's happening I even created a separate branch for this.